libchisel/src/remapstart.rs

```diff
@@ -0,0 +1,181 @@
+use std::collections::HashMap;
+
+use super::{ModuleError, ModulePreset, ModuleTranslator};
```

Comment:
Need to separate the `std`, external crate, and internal imports with one blank line.
*pedantry intensifies*